src/js/defaultValues.js

```diff
@@ -8,7 +10,18 @@ let chartConfiguration = {
     defaultType: 'Percentage', // [Value|Percentage]
     xTicks: null
 };
+
+const choroplethConfig = {
```

Comment:
I didn't understand this, why did we add this here @goyal1092 ?

Reply:
Required default color ranges if there was no config in profile @emre2038 
Also used if profile is working with older config

Reply:
why can't we use the defaults from 
https://github.com/OpenUpSA/wazimap-ng-ui/blob/948b34cbd996bcbda1c0764679a47cfc91f928b4/src/js/configurations/geography_sa.js#L220

Reply:
I didn't realise there was more then one config file.
In tabular comparison we were using defaultValues.chartConfiguration.
Is there a specific reason we didn't move chartConfiguration in geography_sa.js ?

Reply:
I think the initial purpose was that, defaultValue.js would store the constant values which do not change at all. but geography_sa.js stores the default values for the configurable variables. we can override those values by profile configs. so we need to use the one in geography_sa.js